src/main.ts

```diff
@@ -1,8 +1,13 @@
+import 'reflect-metadata';
 import { NestFactory } from '@nestjs/core';
 import { AppModule } from './app.module';
+import * as bodyParser from 'body-parser';
+import * as cors from 'cors';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
+  app.use(cors());
+  app.use(bodyParser.json());
```

Comment:
NestJS is using `bodyParser.json()` behind the scene, so there is no need to explicitly apply here.  Also, for `app.use(cors())`, you can also use `app.enableCors()`